l2tdevtools/lib/netrcfile.py

```diff
@@ -32,7 +32,7 @@ def _GetGitHubValues(self):
       list[str]: .netrc values for github.com or None.
     """
     if not self._contents:
```

Comment:
Should GN's be GNU's ?

Maybe make entire method more compact (same comment for the other methods that were changed):

```
if self._contents:
  # Note that according to GNU's manual on .netrc file, the credential
  # tokens "may be separated by spaces, tabs, or new-lines".
  if value == 'github.com' and self._values[value_index - 1] == 'machine':
    return self._values[value_index + 1:]

return None
```